cmake/version.cmake

```diff
@@ -0,0 +1,14 @@
+find_package(Git QUIET)
+
+# default version string
+set(LIBGRAPHQLPARSER_VERSION "0.0-dev")
```

Comment:
That's the default value. It extracts the current version (ie, 5.0) from the current Git tag using the command "git describe --tags" and chops off the first character. When you're on a development branch, that's something like "v0.5.0-6-g56d5db7". When you're on a tag, it's 'v0.5.0'.